### PR TITLE
[RA2] Removing outdated comments

### LINKED ISSUE #2271: [RA2] Removing outdated comments

### PR DESCRIPTION
**Pull request by EditKoselak**
Closes #2271

### CODE REVIEW

**Review by tomkivlin — APPROVED** [review]

**Review by rgstori — APPROVED** [review]

File: doc/ref_arch/kubernetes/chapters/chapter04.md [hunk 1]
```diff
@@ -87,37 +87,6 @@ In order for the Kubernetes components to be conformant with the Reference Archi
...
 
-<!--
-> THE BELOW TEXT HAS BEEN COMMENTED AS NEEDS REVIEWING AND REPLACED WITH SPECS IN THE ABOVE TABLE AS PER:
-#1635
-
-
-
-This Reference Architecture also specifies:
-
-- Master nodes must run the following Kubernetes control plane services:
```
Comment (pgoyal01) [inline doc/ref_arch/kubernetes/chapters/chapter04.md:98]: Where is this captured in the requirements?

File: doc/ref_arch/kubernetes/chapters/chapter04.md [hunk 2]
```diff
@@ -87,37 +87,6 @@ In order for the Kubernetes components to be conformant with the Reference Archi
...
-    - kube-apiserver
-    - kube-scheduler
-    - kube-controller-manager
-- Master nodes can also run the etcd service and host the etcd database, however etcd can also be hosted on separate nodes if desired
```
Comment (pgoyal01) [inline doc/ref_arch/kubernetes/chapters/chapter04.md:102]: Where is this captured in the requirements?

File: doc/ref_arch/kubernetes/chapters/chapter04.md [hunk 3]
```diff
@@ -167,38 +136,6 @@ Architecture they must be implemented as per the following specifications:
 
 <p align="center"><b>Table 4-6:</b> Storage Solution Specifications</p>
 
-<!--
-> THE BELOW TEXT HAS BEEN COMMENTED AS NEEDS REVIEWING AND REPLACED WITH SPECS IN THE ABOVE TABLE AS PER:
```
Comment (pgoyal01) [inline doc/ref_arch/kubernetes/chapters/chapter04.md:171]: Where are the statements in this block captured in the requirements?

File: doc/ref_arch/kubernetes/chapters/chapter04.md (hunk 2 above)
Comment (CsatariGergely) [reply doc/ref_arch/kubernetes/chapters/chapter04.md:102]: @pgoyal01 , @EditKoselak  I will crosscheck these with the requirements and:
  - Create an issue and pr if something is missing
  - Comment in this pr once all relevant issues are colsed

File: doc/ref_arch/kubernetes/chapters/chapter04.md (hunk 3 above)
Comment (CsatariGergely) [reply doc/ref_arch/kubernetes/chapters/chapter04.md:171]: All requirements listed in this comment block are captured in ra2.stg.001, ra2.stg.002, ra2.stg.003, ra2.stg.004, ra2.stg.005, ra2.stg.006 and ra2.stg.007.

File: doc/ref_arch/kubernetes/chapters/chapter04.md (hunk 2 above)
Comment (CsatariGergely) [reply doc/ref_arch/kubernetes/chapters/chapter04.md:102]: #2283 is to cover the worker node part.

Comment (CsatariGergely) [reply doc/ref_arch/kubernetes/chapters/chapter04.md:102]: #2284 is to cover the workload part.

Comment (CsatariGergely) [reply doc/ref_arch/kubernetes/chapters/chapter04.md:102]: kubelet part is covered by `ra2.k8s.006`.

Comment (rgstori) [reply doc/ref_arch/kubernetes/chapters/chapter04.md:102]: etcd bullet still missing

**Review by pgoyal01 — APPROVED** [review]

**Pull request merged**